guide/src/chapter/contributors.md

```diff
@@ -0,0 +1,7 @@
+# Contributors
+
+Many thanks to all the people who helped make `punktf` a better tool.
```

Comment:
maybe change wording to "an awesome dotfile manager".
and add yourself to the list?

Reply:
Good suggestion, will do that!